packages/snaps-execution-environments/src/common/endowments/timeout.ava.test.ts

```diff
@@ -0,0 +1,19 @@
+import test from 'ava';
+
+import timeout from './timeout';
+
+test('modifying handler should not be allowed and error should be thrown', (tec) => {
+  const { setTimeout: _setTimeout } = timeout.factory();
+
+  const handle = _setTimeout((param: unknown) => param, 100);
+
+  tec.throws(
```

Comment:
What's `tec`?

Reply:
Good question. The object passed to the test is representing execution context of a test. So, I had to add something for the beginning, `tec` stands for `test execution context`. In AVA tests, in general only `t` is used for that, so it would be like `t.throws(...)`. 

The reason why I'm not using only `t` is because of our ESLint configuration which errors with:
```
ESLint: Identifier name 't' is too short (< 2).(id-length)
```

I got suggestion from security team that we should use only `t` as it is in the all AVA tests and SES tests in SES repo. But, I was not too confident that we would like to change that rule in ESLint to do favor to the tests?

Anyway, this can be open topic before I get ready my next PR where I will add a lot more tests and `harden` stuff, I will have to edit that file anyway, it was required for now to keep AVA running. Then we can finally decide if needed.

If you or rest of the team have decision or strong opinion on this, I will be happy to follow that.